cdap-docs/developer-manual/source/cloud-runtimes/example/index.rst

```diff
@@ -10,5 +10,125 @@
+  }'
+
+The value of the data field is the content of your GCP service account key in string form. 
+
+Create a Profile
+----------------
+
+To create a profile, navigate to the Administration page using the menu at the top right of the CDAP UI.
+From this page, you can view the list of system profiles.
+
+.. figure:: /_images/cloud-runtimes/create-system-profile.png
+  :figwidth: 100%
+  :width: 800px
+  :align: center
+  :class: bordered-image
+
+Click on the button to create a new profile. 
+
+.. figure:: /_images/cloud-runtimes/provisioners.png
+  :figwidth: 100%
+  :width: 800px
+  :align: center
+  :class: bordered-image
+
+Select the Google Cloud Dataproc provisioner.
+
+.. figure:: /_images/cloud-runtimes/provisioner-configure.png
+  :figwidth: 100%
+  :width: 800px
+  :align: center
+  :class: bordered-image
+
+Fill in all configuration fields. For the service account key, specify the name of the secure key that
+was added in the setup part of this walk-through.
+
+Create a Test Pipeline
+----------------------
+
+Click on the green plus button and create a new batch pipeline. Our test pipeline will be a very simple
+one that simply reads a file on GCS and writes to an output directory on GCS. Begin by adding a File source
+and a File sink from the left panel. Connect the source to the sink.
+
+.. figure:: /_images/cloud-runtimes/simple-pipeline-overview.png
+  :figwidth: 100%
+  :width: 800px
+  :align: center
+  :class: bordered-image
+
+Configure the source to read a file from the GCS bucket that you set up earlier.
+
+.. figure:: /_images/cloud-runtimes/simple-pipeline-input.png
+  :figwidth: 100%
+  :width: 800px
+  :align: center
+  :class: bordered-image
+
+Configure the sink to write to a directory that does not yet exist.
+
+.. figure:: /_images/cloud-runtimes/simple-pipeline-output.png
+  :figwidth: 100%
+  :width: 800px
+  :align: center
+  :class: bordered-image
+
+Deploy the pipeline by clicking on the `Deploy` button near the top right of the screen.
+This will bring you to the pipeline overview page. Configure the pipeline to use the
+profile that was created earlier.
+
+.. figure:: /_images/cloud-runtimes/pipeline-profile-manual.png
+  :figwidth: 100%
+  :width: 800px
+  :align: center
+  :class: bordered-image
+
+Note that you can also override any profile setting that was not locked during profile creation.
+
+.. figure:: /_images/cloud-runtimes/pipeline-profile-override.png
+  :figwidth: 100%
+  :width: 800px
+  :align: center
+  :class: bordered-image
+
+Run the pipeline. The pipeline should transition to the `Pending` status while the Cloud Dataproc
+cluster is being provisioned for the run. You can view the Cloud Dataproc section of the GCP Console
+to track progress of the cluster. After the cluster has been provisioned, you will see the pipeline
+status transition to `Starting`, then `Running`, then `Succeeded`. This, of course, assumes that your
+GCP account and your pipeline are properly configured.
```

Comment:
unless you're going to add troubleshooting info, suggest we omit:
This, of course, assumes that your GCP account and your pipeline are properly configured.